src/main/kotlin/snyk/container/YAMLImageExtractor.kt

```diff
@@ -5,10 +5,10 @@ import com.intellij.openapi.project.Project
 import com.intellij.openapi.vfs.VirtualFile
 import com.intellij.psi.PsiFile
 import io.snyk.plugin.findPsiFileIgnoringExceptions
-import java.util.stream.Collectors
 
 object YAMLImageExtractor {
     private val logger = Logger.getInstance(javaClass.name)
+    private val imageRegEx = "(?<=image:)(\\s*[a-zA-Z0-9./\\-_]+)(:)?([a-zA-Z0-9./\\-_]+)?".toRegex()
```

Comment:
Did you craft regex yourself or copied from somewhere? If latter, let's link the source place for maintenance purposes 🙏

Reply:
Crafted myself...

Reply:
Poor you.. 😅